cmd/crank/beta/trace/trace.go

```diff
@@ -85,7 +85,8 @@ Examples:
   # Trace a MyKind resource (mykinds.example.org/v1alpha1) named 'my-res' in the namespace 'my-ns'
   crossplane beta trace mykind my-res -n my-ns
 
-  # Output wide format, showing full errors and condition messages
+  # Output wide format, showing full errors and condition messages, and other useful info 
+  # depending on the target type, e.g. composed resources names for XR/XRCs or image used for XPKGs
```

Comment:
in other places of the help text, we're using the full word like `package` instead of `XPKG`. What do you think of keeping it consistent and use the full words here too? i.e.:
```suggestion
  # depending on the target type, e.g. composed resources names for composite resources or image used for packages
```